### ISSUE TITLE
AP slightly lower than expected

### ISSUE THREAD

**Issue by rkotimi** [pangsu0613/CLOCs#30]
Hi! I follow the instructions in README.md to run the  training and evaluation code, but could not get the expected result. 

**The result of training:**
          Generate output labels...
          validation_loss: 0.1410481701822625
          generate label finished(6.10/s). start eval:

          Car AP@0.70, 0.70, 0.70:
          bbox AP:96.92, 93.65, 90.75
          bev  AP:96.29, 90.22, 87.14
          3d   AP:92.42, 81.08, 77.49
          aos  AP:96.77, 93.15, 90.01

          Car AP@0.70, 0.50, 0.50:
          bbox AP:96.92, 93.65, 90.75
          bev  AP:98.78, 96.04, 93.30
          3d   AP:96.75, 95.93, 93.17
          aos  AP:96.77, 93.15, 90.01
          
          Car coco AP@0.50:0.05:0.95:
          bbox AP:79.38, 73.83, 70.88
          bev  AP:74.76, 69.60, 66.95
          3d   AP:64.88, 58.86, 56.25
          aos  AP:79.24, 73.45, 70.32

**The result of evaluation with the the pretrained model you privided:**
          (pytorch1.2) pioneer2_6@amax:/data3/LZW/CLOCs/second$ python ./pytorch/train.py evaluate --config_path=./configs/car.fhd.config --model_dir=/data3/LZW/CLOCs_result/CLOCs_SecCas_pretrained --measure_time=True --batch_size=1
          Predict_test:  False
          sparse_shape: [  41 1600 1408]
          num_class is : 1
          load existing model
          Restoring parameters from ../model_dir/voxelnet-30950.tckpt
          load existing model for fusion layer
          Restoring parameters from /data3/LZW/CLOCs_result/CLOCs_SecCas_pretrained/fusion_layer-11136.tckpt
          remain number of infos: 3769
          Generate output labels...
          [100.0%][===================>][12.12it/s][05:09>00:00]     
          generate label finished(12.15/s). start eval:
          validation_loss: 0.14429050794231943
          avg example to torch time: 1.568 ms
          avg prep time: 12.087 ms

          Car AP@0.70, 0.70, 0.70:
          bbox AP:96.90, 93.62, 90.73
          bev  AP:96.47, 90.28, 89.19
          3d   AP:92.64, 81.13, 77.59
          aos  AP:96.75, 93.14, 89.99

          Car AP@0.70, 0.50, 0.50:
          bbox AP:96.90, 93.62, 90.73
          bev  AP:96.95, 96.10, 93.36
          3d   AP:97.11, 95.91, 93.23
          aos  AP:96.75, 93.14, 89.99
          
          Car coco AP@0.50:0.05:0.95:
          bbox AP:78.65, 73.76, 71.05
          bev  AP:74.59, 69.66, 67.42
          3d   AP:65.07, 59.21, 56.49
          aos  AP:78.51, 73.40, 70.47

I have seen similar issues asked by others, but I think my problem is not the same with them.  I use the files you provided in google drive. I just change the code according to Common Errors & Solutions and a numpy error in the function _do_coco_style_eval_.

I am confused and  hope you can help me solve this problem. Looking forward to your reply. Thank you very much!

**Comment by rkotimi** [comment]
Maybe I set the _detection_2d_path_ Incorrectly. Just now I change _detection_2d_path_, and run the evaluation code again. The result is right.
Now I am running the training code. If the the result is right as well, I will close this issue myself.
So sorry to interrupt you!
